roles/fio-distributed/templates/configmap.yml.j2

```diff
@@ -57,7 +57,7 @@ data:
 {% endif %}
 {% if job_params|default([])|length %}
 {% for match in job_params %}
-{% if match.jobname_match in job %}
+{% if match.jobname_match == job %}
```

Comment:
As far as I know, this conditional does not support regular expressions. I've searched about how to implement regex conditionals in jinja but I'm not able to find a way yet.

Reply:
@bengland2 I figured out a way to perform regex conditionals in Ansible jinja templates via  the `regex_search` filter. i.e

```jinja2
{% if job | regex_search(match.jobname_match) %}
{% endif %}
```

Reply:
I think we want == exact match because what was happening before is that the "w" in the original yaml would match both write,randwrite,readwrite,... , and that leads to fio exiting if you try to define different parameters for the other job types.    Have to be careful using pattern matching.

Reply:
Yeah, I know the intention of using == instead of in. I was suggesting to use the `regex_search` filter since I noticed that you're configuring some jobname_match with regular expressions such as:
     - jobname_match: "^rw$"  of      - jobname_match: "^write$"

Reply:
you were right, I updated PR, resolving this.